### ISSUE TITLE
tox shows warning in default template output

### ISSUE THREAD

**Issue by kitschen** [ionelmc/cookiecutter-pylibrary#194]
After generating a new project from the template, running `tox` shows the warning below. Should the template be adapted?

```
check run-test: commands[0] | python setup.py check --strict --metadata --restructuredtext
running check
warning: Check: This command has been deprecated. Use `twine check` instead: https://packaging.python.org/guides/making-a-pypi-friendly-readme#validating-restructuredtext-markup
```

**Comment by ionelmc** [comment]
Yeah it's a known problem, I haven't gotten to a solution yet. I don't want have something worse, like #109, just to silence a harmless warning. Ideally there would be some publishing workflow where the dists would be checked like in the PR.

But why is a publishing workflow such a big deal you might ask? Dozens of projects have it right? Well, they don't have platform-specific wheels, so it's easy to publish the only two dists (sdist+wheel) in the same place.
